packages/app/src/debug/DebugFailedTest.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <li
+    data-cy="test-row"
+    class="flex flex-row items-center py-12px pr-12px pl-16px gap-x-2.5 non-italic text-base text-gray-700 font-normal border-b-gray-100 border-b-1px"
+  >
+    <span
+      data-cy="test-title"
+    >
+      {{ testData[0] }}
+    </span>
+    <IconChevronRightSmall
+      size="8"
+      stroke-color="gray-200"
+      fill-color="gray-200"
+    />
+    <span
+      data-cy="test-description"
+    >
+      {{ testData[1] }}
```

Comment:
What about tests that have more than two title segments? For example, a spec might be within two or more nested `describe` blocks, or `describe` > `context` > `it`

Reply:
Or, conversely, a test that's at the top level (just an `it` without a wrapping `describe` would just have one title segment, wouldn't it?

Reply:
Yes, this should handle `n` numbers of title segments.

Reply:
I have made this update on the latest commit

Reply:
THANK YOU GUYS! IPHONE 12 Pro max